src/mame/machine/cdicdic.cpp

```diff
@@ -1021,14 +1018,22 @@ void cdicdic_device::handle_cdic_command()
 	switch (m_command)
 	{
 		case 0x23: // Reset Mode 1
+			LOGMASKED(LOG_WRITES, "%s: cdic_w: Reset Mode 1 command\n", machine().describe_context());
 			if (m_disc_command == 0)
 				init_disc_read(DISC_MODE1);
 			break;
 		case 0x24: // Reset Mode 2
+			LOGMASKED(LOG_WRITES, "%s: cdic_w: Reset Mode 2 command\n", machine().describe_context());
+			if (m_disc_command == 0)
+				init_disc_read(DISC_MODE1);
```

Comment:
Given the code around this, shouldn't this be mode 2 instead?

Reply:
This might have something to do with the fact CHDMan doesn't recognize MODE2 in cue sheets, so all CDI CHDs are in MODE1.

Reply:
> Given the code around this, shouldn't this be mode 2 instead?

No